pay-api/src/pay_api/schemas/schemas/business_info.json

```diff
@@ -15,7 +15,8 @@
         "CP1000"
       ],
       "pattern": "^(.*)$",
-      "minLength": 1
+      "minLength": 1,
+      "maxLength": 20
```

Comment:
If a length greater than 20 is passed in, the database will exception 

`(psycopg2.errors.StringDataRightTruncation) value too long for type character varying(20)`